rpm/clsync.init

```diff
@@ -0,0 +1,49 @@
+#! /bin/sh
+#
```

Comment:
Please add something like:
# author: enmaca <enmaca@hotmail.com>
# license: GPL-3+

if it's possible.

I don't know is that normal for RH/CentOS/Fedora, however it's desirable to bypass any potential license problems. :)